### ISSUE TITLE
CalDAV sync for tasks

### ISSUE THREAD

**Issue by HLFH** [cozy/todos#41]
Hi,

It's maybe time to migrate from ownCloud to CozyCloud. But not before you support **CalDAV sync for tasks**. ownCloud does it: https://github.com/owncloud/tasks Please do it too. Then I could use Tasks on Android and Todos on Cozy without any issue: https://play.google.com/store/apps/details?id=org.dmfs.tasks&hl=fr

Thanks in advance,

**Comment by jsilvestre** [comment]
Hello @HLFH 

Todos is an unofficial application that nobody maintain as of today. I'm afraid your feature request will stay on hold for a while before this application becomes a priority again.

Thank you for your suggestion, and sorry to not be able to match your expectations.

**Comment by Thatoo** [comment]
Is it Todos or Tasky supported by the cozy team? I thought it was Todos...
I agree with @HLFH, this feature would be great! I understand it's not the number one priority but it's the missing point for a professional usage of your quatuor "contact, email, calendar and sync". Working between Android, Cozy and Thunderbird/Lightning.

**Comment by HLFH** [comment]
@Thatoo Yes thanks, you're right, Tasky seems the reboot of Cozy Todos, but it's still an unofficial Cozy app. So it's quite a duplicate of this issue: https://github.com/jsilvestre/tasky/issues/25
